common/kubernetes-membership-scheme/src/main/java/org/wso2/carbon/membership/scheme/kubernetes/api/KubernetesHttpsApiEndpoint.java

```diff
@@ -92,4 +96,8 @@ public boolean verify(String hostname, SSLSession session) {
             HttpsURLConnection.setDefaultHostnameVerifier(dummyHostVerifier);
         } catch (Exception ignored) {}
     }
+
+    private String getServiceAccountToken() throws IOException {
+        return new String(Files.readAllBytes(Paths.get(KubernetesMembershipSchemeConstants.BEARER_TOKEN_FILE_LOCATION)));
```

Comment:
@firzhan Thanks for the pull request! Few questions:
- What would be the reason for reading the Token from a file? 
- How do we refresh the Token when it expires?